<?xml version="1.0" ?>
<sdf version="1.6">
  <model name="plate_model">
    <link name="link">
      <inertial>
        <pose>0 0 0 0 0 0</pose>
        <mass>1000.0</mass>
        <inertia>
          <ixx>0.083</ixx>
          <ixy>0.0</ixy>
          <ixz>0.0</ixz>
          <iyy>0.083</iyy>
          <iyz>0.0</iyz>
          <izz>0.083</izz>
        </inertia>
      </inertial>
      <collision name="collision">
        <geometry>
          <mesh>
            <uri>model://plate_model/meshes/COLLISION_plate_model.stl</uri>
            <!-- 600 mm x 450 mm -->
            <scale>0.001 0.001 0.001</scale>
          </mesh>
        </geometry>
      </collision>
      <visual name="visual">
        <geometry>
          <mesh>
            <uri>model://plate_model/meshes/plate_model.stl</uri>
            <!-- 600 mm x 450 mm -->
            <scale>0.001 0.001 0.001</scale>
          </mesh>
        </geometry>
        <material> <!-- dark material -->
          <ambient>0 0 0 1</ambient>
          <diffuse>0 0 0 1</diffuse>
          <specular>0 0 0 0</specular>
          <emissive>0 0 0 1</emissive>
        </material>
      </visual>
    </link>
  </model>
</sdf>